modules/Dasboard/components/ClassroomFormModal/ClassroomFormModal.helpers.ts

```diff
@@ -0,0 +1,20 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+
+export const ClassroomFormSchema = z.object({
+  title: z.string().min(1, { message: "Title is required" }),
+  subject: z.string().min(1, { message: "Subject is required" }),
+  class_time: z.string().min(1, { message: "Class time is required" }),
+  days: z.array(z.string()).min(1, { message: "At least one day is required" }).optional(),
```

Comment:
Should this be an optional field?

Reply:
No bhaiya, there were some error so I used that but later I forgot to remove it.